### PR TITLE
[bugfix] TF-2465 Blue bar parsing failed for the time of event

### PR DESCRIPTION
**Pull request by hieutbui**
## Issue
---
#2465 

## Resolved
---

https://github.com/linagora/tmail-flutter/assets/80142234/d023b5bf-b230-47b8-afad-2110915d1964

### CODE REVIEW

File: test/features/email/presentation/extensions/calendar_event_extension_test.dart [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+void main() {
+  group('calendar event extension text', () {
+    final dateTime = DateTime(2022, 12, 31, 13, 39, 00, 00, 00);
+    final sundayInCurrentDateLocale = AppUtils.getCurrentDateLocale().daysLong[5];
+    final decemberInCurrentDateLocale = AppUtils.getCurrentDateLocale().monthsLong.last;
+    final meridiemIndicator = AppUtils.getCurrentDateLocale().pm;
+    final expectedFormattedDateTime = '$sundayInCurrentDateLocale, $decemberInCurrentDateLocale 31, 2022 01:39 $meridiemIndicator';
+    final expectedFormattedTime = '01:39 $meridiemIndicator';
```
Comment (dab246) [inline test/features/email/presentation/extensions/calendar_event_extension_test.dart:13]: Expected value should leave a specific value

Comment (hieutbui) [reply test/features/email/presentation/extensions/calendar_event_extension_test.dart]: done

**Review by dab246 — COMMENTED** [review]
please rebase it

**Review by dab246 — APPROVED** [review]
LGTM

**Review by hoangdat — APPROVED** [review]

**Pull request merged**